android/src/main/java/com/swmansion/reanimated/ReanimatedJSIModulePackage.java

```diff
@@ -1,32 +0,0 @@
-package com.swmansion.reanimated;
-
-import android.util.Log;
-import com.facebook.react.bridge.JSIModulePackage;
-import com.facebook.react.bridge.JSIModuleSpec;
-import com.facebook.react.bridge.JavaScriptContextHolder;
-import com.facebook.react.bridge.ReactApplicationContext;
-import java.util.Arrays;
-import java.util.List;
-
-public class ReanimatedJSIModulePackage implements JSIModulePackage {
-
-  @Override
-  public List<JSIModuleSpec> getJSIModules(
```

Comment:
One thing that I think of now is that we can actually make the transition less of a pain and keep this class with empty implementation + some warning message. This way people won't get compile errors after upgrading. We can show a yellow box for them to update MainApplication.java

Reply:
I added `@Deprecation` annotation and `Log.w()`. `LogBox` doesn't have a native interface so we'd need to set some global variable or emit an event which isn't ideal.